barretenberg/cpp/src/barretenberg/dsl/acir_proofs/honk_zk_contract.hpp

```diff
@@ -0,0 +1,1891 @@
+#pragma once
+#include "barretenberg/honk/utils/honk_key_gen.hpp"
+#include <iostream>
+
+// Source code for the Ultrahonk Solidity verifier.
+// It's expected that the AcirComposer will inject a library which will load the verification key into memory.
+// NOLINTNEXTLINE(cppcoreguidelines-avoid-c-arrays)
+static const char HONK_ZK_CONTRACT_SOURCE[] = R"(
+
+pragma solidity ^0.8.27;
```

Comment:
This is identical with the BaseZKHonkVerifier in terms of logic but I removed the Base class because the contract was over the allowed bytecode size. We are still very close to the limit so it is going to be a problem again if we add new functionality but ultimately the rewrite to assembly will probably solve this issue. I tried to not harm readability too much.